src/components/primitives/Download/Download.md

```diff
@@ -2,6 +2,7 @@
 import { GlobalThemeProvider } from "../../../theme/provider";
 import ImageModal from "../../../styleguide/ImageModal";
 import img1 from "./Download.png";
+import Download from "./Download";
 
 <GlobalThemeProvider>
```

Comment:
the description should display a component in all possible themes (see other examples) 


```tsx
import { DefaultTheme, ThemeProvider } from "styled-components";
import themes from "../../../theme";


<React.Fragment>
  {Object.entries(themes).map((theme) => (
    <ThemeProvider theme={theme[1]} key={theme[0]}>
        { // your component here }
    </ThemeProvider>
)}
</React.Fragment>
```

Reply:
by in this case Download component has only mode type (light and dark), there is no difference in theme